web/lattice_surgery_web/settings.py

```diff
@@ -79,12 +79,12 @@
 # Database
 # https://docs.djangoproject.com/en/3.1/ref/settings/#databases
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
-    }
-}
+# DATABASES = {
+#     'default': {
+#         'ENGINE': 'django.db.backends.sqlite3',
+#         'NAME': BASE_DIR / 'db.sqlite3',
+#     }
+# }
```

Comment:
Gettin the error:
```
[...]
  File "/var/www/vhosts/lattice-surgery-compiler/webgui/env/lib64/python3.8/site-packages/django/db/utils.py", line 111, in load_backend
    return import_module('%s.base' % backend_name)
  File "/usr/lib64/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/var/www/vhosts/lattice-surgery-compiler/webgui/env/lib64/python3.8/site-packages/django/db/backends/sqlite3/base.py", line 70, in <module>
    check_sqlite_version()
  File "/var/www/vhosts/lattice-surgery-compiler/webgui/env/lib64/python3.8/site-packages/django/db/backends/sqlite3/base.py", line 67, in check_sqlite_version
    raise ImproperlyConfigured('SQLite 3.8.3 or later is required (found %s).' % Database.sqlite_version)
django.core.exceptions.ImproperlyConfigured: SQLite 3.8.3 or later is required (found 3.7.17).
```

Reply:
@Keelando

Reply:
We could use mysql in the future

Reply:
Or there is this solution https://stackoverflow.com/questions/56520658/djangoshowing-improperlyconfigured-error-sqlite-3-8-3-or-later-is-required-f

Reply:
Spent a lot of time on this. Tried to run a custom version of sqlite, but in some cases the versions still don't match. Will have to either:
- Build python from scratch
- Downgrade django
- Containerize the application

It is a well known bug

Reply:
Or fourth option one can insist with a custom compiled sqlite https://stackoverflow.com/questions/55674176/django-cant-find-new-sqlite-version-sqlite-3-8-3-or-later-is-required-found

Reply:
Hmm. I'm running sqlite3 version 2.6.0. Try pip install sqlite3?

Reply:
```
ERROR: Could not find a version that satisfies the requirement sqlite3>=2.6.0 (from -r requirements.txt (line 127)) (from versions: none)
ERROR: No matching distribution found for sqlite3>=2.6.0 (from -r requirements.txt (line 127))
```
It seems that sqlite3 is not a package for python 3.8